### DISCUSSION TITLE
Generic (non-Asus) use cases

### DISCUSSION THREAD

**Discussion by cjb-eng** [osy/XG_Mobile_Station#9] [Hardware]
I don't own the ROG Ally or a Flow product so I don't have an appreciation for some of the subtle details in using those hardware components. My interest in this project is for more generic applications (e.g. eGPU for my laptop). From what I've been reading, it sounds like Asus has maybe broke away from the Thunderbolt standard so the XG_Mobile_Dock might not be compatible with non-Asus hardware, is that accurate?

**Comment by FullstackSensei** [comment]
Yes. The XG mobile has nothing to do with Thunderbolt. If you want Thunderbolt eGPU enclosure, get a Razer Core enclosure. They're quite cheap and support almost any GPU. I have one and the 3090 FE fits in comfortably.

**Comment by cjb-eng** [comment]
Agreed the Razer is the commercial solution. But the allure here was the
open source project  and a perceived opportunity to get more intimate
familiarity with Thunderbolt technology - I won't get that kind of insight
from a COTS product...

In the spirit of furthering this open source development, is there any possibility of mod'ing the Lite variant as it was designed as a drop-in replacement for the [ADT-UT3G](https://www.adt.link/product/UT3G.html). The ADT-UT3G does have on-board Thunderbolt tech. The Lite variant was intended as a ADT-UT3G drop-in but reviewing it's schematic its clear it lacks many of the hardware components of the ADT-UT3G. Is there a consensus that mod'ing the Lite variant as a fully-featured ADT-UT3G to operate as a generic eGPU is a dead-end because, say, it requires licensed components?
